### ISSUE TITLE
Accessibility: long-press FAB menu foes not get focus with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [thunder-app/thunder#580]
**Description**  
when the FAB is long pressed and the options appears the talkback focus should be pulled to the first element ("back to top by default)

**How to Reproduce**  
1. Go to system settings > accessibility > talkback and turn it on
2. Open thunder
3. Double tap and hold FAB to open its options

**Expected Behavior**  
Talkback focus should be pulled to the top item in the list of actions

**Screenshots**  

**Device & App Version:**  
 - Device: [e.g. iPhone6] OnePlus Nord N200
 - OS: [e.g. iOS8.1] Android 13
 - Version [e.g. 22] v0.2.1-alpha.2+14

**Additional Context**
